package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "spotamovie-be",
+  "version": "1.0.0",
+  "description": "recommendation for movies",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Start testing as soon as you can. No change requested in this PR, but I want it in the next one.